src/js/pages/catalog/PackagesTab.js

```diff
@@ -218,102 +239,130 @@ class PackagesTab extends mixin(StoreMixin) {
 
   getCommunityPackagesGrid(packages) {
     const isSearchActive = this.state.searchString !== "";
-    if (!isSearchActive && packages.getItems().length === 0) {
-      return null;
-    }
-
-    let subtitle = (
-      <Trans render="p" className="tall flush-top">
-        Community packages are unverified and unreviewed content from the
-        community.
-      </Trans>
-    );
-    let title = <Trans render="span">Community</Trans>;
     const titleClasses = classNames("flush-top", {
       short: !isSearchActive,
       tall: isSearchActive
     });
 
-    if (isSearchActive) {
-      const foundPackagesLength = packages.getItems().length;
-      if (foundPackagesLength < 1) {
-        return (
-          <Trans render="div" className="clearfix">
-            No results were found for your search: "{this.state.searchString}" (
-            <a className="clickable" onClick={this.clearInput}>
-              view all
-            </a>
-            )
-          </Trans>
-        );
-      }
-
-      // L10NTODO: Pluralize
-      const packagesWord = StringUtil.pluralize("service", foundPackagesLength);
-      subtitle = null;
-      title = (
-        <Trans render="span">
-          {packages.getItems().length} {packagesWord} found
-        </Trans>
-      );
-    }
-
     return (
       <div className="clearfix">
-        <h1 className={titleClasses}>{title}</h1>
-        {subtitle}
+        <h1 className={titleClasses}>
+          <Trans render="span">Community</Trans>
+        </h1>
+        <Trans render="p" className="tall flush-top">
+          Community packages are unverified and unreviewed content from the
+          community.
+        </Trans>
         <div className="panel-grid row">{this.getPackageGrid(packages)}</div>
       </div>
     );
   }
 
+  getFilterButtons(selectedLength, communityLength) {
+    const numbers = {
+      all: selectedLength + communityLength,
+      certified: selectedLength,
+      community: communityLength
+    };
+
+    const currentFilter = this.state.searchFilter;
+
+    const buttons = Object.entries(numbers).map(([filter, number]) => {
+      const classSet = classNames("button button-outline", {
+        active: filter === currentFilter
+      });
+
+      const name = StringUtil.capitalize(filter);
+      return (
+        <button
+          key={filter}
+          className={classSet}
+          onClick={this.getSearchFilterChangeHandler(filter)}
+        >
+          <Trans render="b" id={name}>
```

Comment:
if your using the `id` prop then you dont need to put content inside the `Trans` component. This should be written `<Trans render="b" id={name} />` using `{name}` twice is redundant.

Reply:
Thanks, I will change it.